packages/wonder-blocks-dropdown/components/single-select.md

```diff
@@ -301,3 +301,31 @@ const styles = StyleSheet.create({
     <SingleSelect menuText="Empty" placeholder="empty" />
 </View>
 ```
+
+### Accessibility
+
+If you need to associate this component with another element (e.g. `<label>`),
+make sure to pass the `aria-labelledby` and/or `id` props to the `SingleSelect` component.
```

Comment:
Given that we'll want to add these props to every component eventually I think we should probably just do that.  Having examples for each component to show that they handle common props seems like a lot of overhead.  Having some tests that we can automatically determine if `id`, `aria-labelledby`, etc. got set on a particular element within a component may be a more sustainable approach.

It would be good to have a top-level .md file for the whole repo that documents what the common props are on our components.  Not sure how to make that work with styleguidist though.  This is something for a separate PR though.

Reply:
That's a great point, I'm going to create a separate ticket to work on that.

Reply:
https://khanacademy.atlassian.net/browse/WB-583

Reply:
Follow up diff where ariaProps are added to Wonderblocks components: https://github.com/Khan/wonder-blocks/pull/411#pullrequestreview-245110722